compiler.rst

```diff
@@ -10,16 +10,16 @@ Abstract
 Historically (through 2.4), compilation from source code to bytecode
 involved two steps:
 
-1. Parse the source code into a parse tree (Parser/pgen.c)
-2. Emit bytecode based on the parse tree (Python/compile.c)
+1. Parse the source code into a parse tree (:file:`Parser/pgen.c`)
+2. Emit bytecode based on the parse tree (:file:`Python/compile.c`)
```

Comment:
Hm, just to make sure I do the correct amendment, are you suggesting I drop *all* historical references? In short, the section can start out with:

> In CPython, the compilation from source code to bytecode involves three steps:
>   1. Parse source code into a parse tree (Parser/pgen.c)		 
>   2. Transform parse tree into an Abstract Syntax Tree (Python/ast.c)		 
>   3. Transform AST into a Control Flow Graph (Python/compile.c)		 
>   4. Emit bytecode based on the Control Flow Graph (Python/compile.c)	
>	
> The purpose of this document is to outline how these three steps of the process work.
  		  
and continue with the rest as-is?